### ISSUE TITLE
Create files base64 and sed errors

### ISSUE THREAD

**Issue by unaizorrilla** [githubtraining/training-manual#345]
Hi, 

Reviewing the material for my acredition I discover two minor issues on create_files script, the first one is about base64 function, because on my machine ( WSL2 ) the $content generate a return line and the post file fail (400 bad request ) because invalid json.

The sed ejectued on non msys plattform is also malformed also (blank quotes)  I fix both issues as I show on the code

```bash
        # Insert slide image into slide
        sed -i "s|IMAGE|$random_image|g" "$new_file" >>log.out 2>&1
        # Encode the file
        content=$(base64 -w 0 <"$new_file")
```

**Comment by unaizorrilla** [comment]
If you are interested I can create PR with my changes,

**Comment by unaizorrilla** [comment]
Well... another bug more on the scripts

 - on shared_functions the --quiet parameter for http is also not supported!

**Comment by parkerbxyz** [comment]
Thanks for reporting this, @unaizorrilla. Feel free to open a pull request with your changes and we’ll review it.

**Comment by pamontag** [comment]
> Well... another bug more on the scripts
> 
> * on shared_functions the --quiet parameter for http is also not supported!

Hello, I had the same issue but the --quiet parameter is not supported for httpie 1.0.3 version that is shipped automatically on Ubuntu using apt-get in my environment. I solved installing httpie version 3.0.2 using snapd package manager but is not automatically managed by bootstrap script.

**Comment by amyschoen** [comment]
@pamontag, I ran into a similar issue around HTTPie version and filed #344.

**Comment by parkerbxyz** [comment]
This issue will be automatically closed when #351 is merged. If the sed issue isn’t resolved, feel free to re-open this issue or create a new one.
